manticore/platforms/evm.py

```diff
@@ -863,6 +871,18 @@ def constraints(self, constraints):
     def gas(self):
         return Operators.EXTRACT(self._gas, 0, 256)
 
+    def jumpi_false_branch(self) -> Optional[int]:
```

Comment:
Python has a nifty replacement for getter & setter methods called [Properties](https://docs.python.org/3/library/functions.html#property). Since it looks like these don't explicitly need to be callable, we should convert them to properties.

Reply:
Fixed.